<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoom</class>
 <widget class="QDialog" name="ChatRoom">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="chatUserLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>picture/close.jpg</normaloff>picture/close.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="sendFileBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>发送文件</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>picture/file.jpg</normaloff>picture/file.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="RightWidget" native="true">
   <property name="geometry">
    <rect>
     <x>819</x>
     <y>30</y>
     <width>171</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendImgBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>发送图片</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>picture/Img.jpg</normaloff>picture/Img.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="chatContentList">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>651</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="screenShotBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>截图(Ctrl+Alt+f)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>picture/screenShot.jpg</normaloff>picture/screenShot.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="chatUserList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>171</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>最小化</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>picture/min.jpg</normaloff>picture/min.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatTextEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>651</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMsgBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>530</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="toolTip">
    <string>选择字体</string>
   </property>
   <property name="currentText">
    <string>Ubuntu</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fontSizeComboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>字体大小</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
